ui-tests/src/main/java/io/jenkins/plugins/analysis/warnings/AbstractIssuesTableRow.java

```diff
@@ -0,0 +1,23 @@
+package io.jenkins.plugins.analysis.warnings;
+
+/**
+ * Abstract representation of a table row of the issues-table.
+ *
+ * @author Stephan Plöderl
+ */
+public abstract class AbstractIssuesTableRow {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This abstract class does not have any abstract methods](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5151082)